lib/mix/tasks/thrift.generate.ex

```diff
@@ -6,53 +6,61 @@ defmodule Mix.Tasks.Thrift.Generate do
   @moduledoc """
   Generate Elixir source files from Thrift schema files (`.thrift`).
 
-  A list of files can be given after the task name in order to select the
-  specific Thrift schema files to parse:
+  A list of files should be given after the task name in order to select
+  the specific Thrift schema files to parse:
 
       mix thrift.generate file1.thrift file2.thrift
 
   ## Command line options
 
-    * `-o` `--out` - set the output directory, overriding the `:output_path`
-      configuration value
-    * `-v` `--verbose` - enable verbose task logging
+    * `-I dir` / `--include dir` - add a directory to the list of
```

Comment:
It's the former, which fortunately matches the way that the `thrift` compiler works. I'll expand the description to make it clear that this option can be repeated.